api/client/image/build.go

```diff
@@ -211,6 +227,9 @@ func runBuild(dockerCli *client.DockerCli, options buildOptions) error {
 
 	// Setup an upload progress bar
 	progressOutput := streamformatter.NewStreamFormatter().NewProgressOutput(progBuff, true)
+	if !dockerCli.IsTerminalOut() {
```

Comment:
@tonistiigi No a problem. The pull request has been updated. And thanks for spending time to review the PR. 😃